web/src/pages/createStaff/index.page.jsx

```diff
@@ -0,0 +1,46 @@
+import { Button } from 'flowbite-react';
+import React from 'react';
+
+import TextInput from '@/components/organisms/TextInput';
+
+
+//webpage backdround
+const backgroundColor = () => {
+  return (
+    <div className="bg-white min-h-screen p-4"> {/* Use Flowbite class for white background */}
+      
+
+      <form className="mt-10 ml-4 w-1/2 lg:w-1/4  ">
+        
+         <div className>
+         <TextInput label='Name' name='createStaff' />
+             </div>
+
+                  <div className = 'mt-5'>
+               <TextInput label='Position' name='createStaff' />
+                  </div>
+
+                        <div className ='mt-5'>
+                           <TextInput label='Contact Number' name='createStaff' />
+                              </div>
+
+                                 <div className ='mt-5'>
+                                       <TextInput label='Account Type' name='createStaff' />
+                                                </div> 
```

Comment:
1. IMPORTANT: Please observer proper indention please
2. div mt-5 is not needed, it should be handled in the the wrapper/parent/form component
```suggestion
         <TextInput label='Name' name='createStaff' />
         <TextInput label='Position' name='createStaff' />
         <TextInput label='Contact Number' name='createStaff' />
         <TextInput label='Account Type' name='createStaff' />
```